scripts/automate.py

```diff
@@ -133,19 +132,19 @@ def _generate_front_matter(
 
     :param snakemd.Document doc: the document to add the front matter to.
     :param str title: the title of the document.
-    :param datetime.datetime created_at: optional date/time when item is created.
-    :param datatime.datetime last_modified: optional date/time when item was last modified.
+    :param Optional[List[Optional[datetime.datetime]]] times: optional list of
+        date/times that may be `None`.
```

Comment:
I quite like this change! Having all the times could be potentially useful later for other features.